libraries/AP_HAL_Linux/AP_HAL_Linux_Namespace.h

```diff
@@ -30,6 +30,7 @@ namespace Linux {
     class LinuxRCOutput_AioPRU;
     class LinuxRCOutput_Navio;
     class LinuxRCOutput_ZYNQ;
+    class LinuxRCOutput_Bebop;
```

Comment:
Is this RCOutput exclusive to Bebop?  Maybe it's used in future drones? Shouldn't it be better to name after the chip/implementation rather the board it's used in? I know right now some of our classes are named after the board, but with the increasing number of Linux boards this is going to change.  Do you have any other name in mind?